modules/aleph/includes/aleph.reservation.inc

```diff
@@ -146,16 +149,16 @@ function aleph_reservation_default_interest_period($profile = NULL) {
  * Provider list callback.
  *
  * @param object $account
- *    User to list reservations for.
+ *   User to list reservations for.
  * @param string|null $requested_type
- *    Type of reservations to list. NULL for all.
+ *   Type of reservations to list. NULL for all.
  *
  * @return \DingProviderReservation[]
- *    Array of DingProviderReservations.
+ *   Array of DingProviderReservations.
  */
 function aleph_reservation_list($account, $requested_type = NULL) {
   // Throw exception if we're not logged in.
-  ding_user_get_creds($account);
+  ding_user_get_creds();
```

Comment:
Is there a reason why we are not passing the `$accout` parameter anymore?

Reply:
It's not used by ding_user_get_creds().

Reply:
:+1: Gah. Good point!